### ISSUE TITLE
helm-mini: void-variable helm-buffers-truncate-line

### ISSUE THREAD

**Issue by manuel-uberti** [emacs-helm/helm#1058]
Hi,

I've just upgraded to the latest version of Helm on MELPA. When I call ```helm-mini``` with ```toggle-debug-on-error```, this is what I get:
```
Debugger entered--Lisp error: (void-variable helm-buffers-truncate-line)
  helm-mini()
  funcall-interactively(helm-mini)
  call-interactively(helm-mini record nil)
  command-execute(helm-mini record)
  helm-M-x(nil "helm-mini")
  funcall-interactively(helm-M-x nil "helm-mini")
  call-interactively(helm-M-x nil nil)
  command-execute(helm-M-x)
```
If you need me to do other tests, please feel free to ask.

**Comment by xuchunyang** [comment]
Fixed in #1056, you need wait for about 1 hour to upgrade and MELPA will build latest helm.

**Comment by manuel-uberti** [comment]
Thanks.
